app/routes/user/endorsements.tsx

```diff
@@ -215,49 +222,101 @@ export function EndorsementRequestCard({
 }) {
   const fetcher = useFetcher()
   const disabled = fetcher.state !== 'idle'
+  const [showMore, setShowMore] = useState(false)
+  const [isOverflowing, setIsOverflowing] = useState(true)
+  const ref = useRef<HTMLDivElement>(null)
+
+  function overflowing() {
+    if (ref.current)
+      setIsOverflowing(ref.current?.offsetWidth < ref.current?.scrollWidth)
+  }
```

Comment:
There's really no simple way to do this with CSS?

Reply:
Not really, but it does work as expected